pylot/perception/tracking/object_tracker_operator.py

```diff
@@ -137,6 +149,10 @@ def __track_bboxes_on_frame(self, camera_frame, timestamp, catch_up):
             # The tracker must be reinitialized.
             self._ready_to_update = False
         else:
+            # Get runtime in ms.
+            runtime = (time.time() - start_time) * 1000
+            obstacle_tracking_stream.send(
+                ObstaclesMessage(timestamp, tracked_obstacles, runtime))
```

Comment:
In another PR we'll have to change the type of the output message. We should send https://github.com/erdos-project/pylot/blob/master/pylot/perception/messages.py#L202 such that the downstream operators (i.e., prediction) can use the past trajectories to predict future trajectories.

We can either do it here, or alternatively insert an operator that converts ObstaclesMessages to ObstaclesTrajectoriesMessages. IMHO, the second option may be more desirable because: 1) we can keep the tracker output in 2D in case we want to do some other processing, and 2) the new operator will provide a configurable way of transforming from 2D to 3D (e.g., using point cloud, depth estimation).

Reply:
Can you please open an issue?